04_fep/05_ATM/01-ATM.ipynb

```diff
@@ -0,0 +1,481 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5b0da21b-4468-4dcd-8138-4bd0d51620c6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "solvated = BSS.Solvent.tip3p(molecule=system, box=3 * [7 * BSS.Units.Length.nanometer])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ecaa0ab-4436-43f7-b211-3529b9c78b84",
+   "metadata": {},
+   "source": [
+    "# Minimisation and Equilibration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e921c870-cb23-4b9e-b358-260f64b6218b",
+   "metadata": {},
+   "source": [
+    "ATM simulations require a significant number of restraints, as well as the gradual introduction of the ATMForce itself. As such standard minimisation and equilibration protocols are not appropriate and a series of custom protocols is needed. \n",
+    "\n",
+    "In this section we will cover a full minimisation and equilibration protocol for the ATM system we created above. We won't actually run any simulations as they would take far too long, but we will still cover each unique step in the protocol.\n",
+    "\n",
+    "ATM simulations are best run with a series of restraints:\n",
+    "- __core alignment__; these are the the rigid core restraints that are applied to the atoms we found earlier.\n",
+    "- __positional restraints__; these are flat-bottom restraints used to keep specific atoms in place, these are generally applied to the alpha carbons of the protein.\n",
+    "- __centre of mass distance restraints__; these are used to maintain the relative positions of the protein and ligands and are applied to the centre off mass of the atoms listed in the `data` dictionary (these were found automatically when we prepared the system, but they can be set manually if the ones found are not appropriate).\n",
+    "\n",
+    "Now let's find the alpha carbons that we will be restraining using BioSimSpace search functionality."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8727fdbb-305f-459a-b8c6-eb5a77424b36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ca = [atom.index() for atom in solvated.search(\"atomname CA\")]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ab9036b-5cd8-40ee-b1e4-38dff16eb9d4",
+   "metadata": {},
+   "source": [
+    "Now we can create a minimisation protocol using the forces and `atm_data` dictionary we created when setting up the system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1de322e-f178-444e-bfc7-8f5936182b96",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "minimisation = BSS.Protocol.ATMMinimisation(\n",
+    "    data=atm_data, core_alignment=True, restraint=ca, com_distance_restraint=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "56b04fa5-aa1e-4032-9ca6-e79a21240845",
+   "metadata": {},
+   "source": [
+    "This protocol can then be run using the BioSimSpace openMM runner."
```

Comment:
update the comment to mention that this may take a few minutes to complete. 
do we need to run by default for this tutorial with the default choice of step=10000?

Reply:
Minimisation now cut to just 100 steps, takes around 100 seconds on the try OBS server.

Reply:
Testing minimisation and it seems that ~200 steps is the minimum to give a system that is stable going forward. Added a comment to say it may take a few minutes.